app/views/user/index.html.erb

```diff
@@ -1 +1,25 @@
-<h1>User Index HTML File</h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>"Blog Website"</title>
+    <%= stylesheet_link_tag "user/index",
+    media: "all",
+    "data-turbolinks-track": "reload" %>
+  </head>
+  <body>
+
+    <div class="subParentContent">
+       <% @users.each do |user| %>
```

Comment:
- [ ] It would be better if we add a link so that we can click on any user and navigate to their show page
![Screenshot from 2023-02-03 00-38-06](https://user-images.githubusercontent.com/46388113/216481934-64a52bac-0c74-4757-9efd-4c8a59ee5d7c.png)